CHANGELOG.md

```diff
@@ -20,6 +20,25 @@
 
   ([PR #N](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/N))
 
+- New component: Digital Marketplace Footer component
+
+  *Requirement before using this component:*
+  - To use this the application must be using `govuk-frontend` template
+  - digitalmarketplace-util v48.8.0+ must be installed
```

Comment:
Ta for fixing - if you want, here is the link to the change: https://github.com/alphagov/digitalmarketplace-utils/commit/c62eed3350438ed057e2138b6757a9579ce4feb6